apps/frontend/src/components/global/upload_tabs/DatabaseReader.vue

```diff
@@ -87,6 +98,10 @@ export default class DatabaseReader extends mixins(ServerMixin) {
       });
   }
```

Comment:
This is probably more suited to live in the Evaluations store now since this.files was moved there.